### PR TITLE
Bug 1504729 - Log job state when getting last op

### PR DESCRIPTION
**Pull request by djzager**
Simple enhancement so that we can see the state of the last operation when the service-catalog requests it.

### CODE REVIEW

File: pkg/broker/broker.go [hunk 1]
```diff
@@ -1002,9 +1002,9 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
 
 		if async, provision: it should create a Job that calls apb.Provision. And write the output to etcd.
 	*/
-	a.log.Debug(fmt.Sprintf("service_id: %s", req.ServiceID)) // optional
-	a.log.Debug(fmt.Sprintf("plan_id: %s", req.PlanID))       // optional
-	a.log.Debug(fmt.Sprintf("operation:  %s", req.Operation)) // this is provided with the provision. task id from the work_engine
+	a.log.Debugf("service_id: %s", req.ServiceID)
```
Comment (eriknelson) [inline pkg/broker/broker.go]: Thanks for killing the fmt.Sprint, there's a lot of that left over from the old days :)

File: pkg/broker/broker.go [hunk 2]
```diff
@@ -1002,9 +1002,9 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
...
+	a.log.Debugf("plan_id: %s", req.PlanID)
+	a.log.Debugf("operation:  %s", req.Operation) // this is provided with the provision. task id from the work_engine
```
Comment (eriknelson) [inline pkg/broker/broker.go]: Nit: the comment is a little inaccurate, it'll be there with every async op, not just provision.

File: pkg/broker/broker.go [hunk 3]
```diff
@@ -1004,7 +1004,7 @@ func (a AnsibleBroker) LastOperation(instanceUUID uuid.UUID, req *LastOperationR
 	*/
 	a.log.Debugf("service_id: %s", req.ServiceID)
 	a.log.Debugf("plan_id: %s", req.PlanID)
-	a.log.Debugf("operation:  %s", req.Operation) // this is provided with the provision. task id from the work_engine
```
Comment (jmrodri) [inline pkg/broker/broker.go]: Why is this an erroneous comment? The operation is expected to be the job token (task_id) from the work_engine that happened during the provision. At a minimum I'd remove provision and make it clear that we expect operation to be the job token id.

Comment (eriknelson) [reply pkg/broker/broker.go]: I just remarked it wasn't entirely accurate since all async jobs will set that to be the job token, not just provision.

**Comment by jmrodri** [comment]
If this was changing something in docs or build file stuff I would let it slide. This is actually changing mainline broker code so a bugzilla is required.

**Comment by rthallisey** [comment]
https://github.com/openshift/ansible-service-broker/issues/475

**Pull request merged**